networkit/test/test_graph.py

```diff
@@ -253,5 +254,10 @@ def testAddEdgeWithMissingNodes(self):
 		self.assertEqual(G.numberOfNodes(), 3)
 		self.assertEqual(G.numberOfEdges(), 2)
 
+	def testGraphPickling(self):
+		G = nk.Graph(2)
+		G.addEdge(0,1)
+		pickle.dumps(G)
```

Comment:
The test should also verify that unpickling works and that the deseralized graph is identical to the serialized one.